src/Umbraco.Core/Services/PackagingService.cs

```diff
@@ -1005,7 +1005,7 @@ private EntityContainer CreateDataTypeChildFolder(string folderName, IUmbracoEnt
             var found = children.Any(x => x.Name.InvariantEquals(folderName));
             if (found)
             {
-                var containerId = children.Single(x => x.Name.InvariantEquals(folderName)).Id;
+                var containerId = children.SingleOrDefault(x => x.Name.InvariantEquals(folderName)).Id;

```

Comment:
this change would also have the potential to throw a NullReferenceException due to not checking for null before attempting to access the Id property...  If you want to make this change it will need another check in place.